README.md

```diff
@@ -1,39 +1,72 @@
+TODO: add azure/nuget badges
 
-linq2db.LINQPad is a driver for [LINQPad](http://www.linqpad.net) that supports the following databases:
+# LINQ to DB LINQPad Driver
 
-- **DB2** (LUW, z/OS) *(client software must be installed)*
-- **DB2 iSeries** (using [3rd-party provider](https://github.com/LinqToDB4iSeries/Linq2DB4iSeries)) *(iAccess 7.1+ software must be installed)*
+linq2db.LINQPad is a driver for [LINQPad](http://www.linqpad.net). Supported versions: LINQPad 5 and LINQPad 6.
+
+Following databases supported (by both LINQPad 5 and LINQPad 6 if not noted):
+
+- **DB2** (LUW, z/OS)
+- **DB2 iSeries** (using [3rd-party provider](https://github.com/LinqToDB4iSeries/Linq2DB4iSeries)) *(iAccess 7.1+ software must be installed)*. **IMPORTANT:** currently available only for LINQPad 5 using lin2db.LINQPad version 2.9.3 or earlier
 - **Firebird**
-- **Informix** *(client software must be installed)*
-- **Microsoft Access**
-- **Microsoft Sql Azure**
-- **Microsoft Sql Server** 2000+
-- **Microsoft SqlCe**
-- **MySql**
-- **Oracle** *(client software must be installed if native driver is used)*
+- **Informix**
+- **Microsoft Access** *(supports both OleDb and ODBC)*
+- **Microsoft Sql Server** 2000+ *(including **Microsoft Sql Azure**)*
+- **Microsoft Sql Server Compact (SqlCe)**
+- **MySql/MariaDB**
+- **Oracle**
 - **PostgreSQL**
 - **SQLite**
-- **SAP HANA** *(client software must be installed)*
-- **SAP/Sybase ASE** *(client software must be installed)*
+- **SAP HANA** *(client software must be installed, supports both Native and ODBC providers)*
+- **SAP/Sybase ASE**
 
 ## Download
 
-Releases are hosted on [Github](https://github.com/linq2db/linq2db.LINQPad/releases).
+Releases are hosted on [Github](https://github.com/linq2db/linq2db.LINQPad/releases) and on [Nuget](https://www.nuget.org/packages/linq2db.LINQPad) for LINQPad 6 driver.
+
+Latest build is hosted on [Azure Artifacts](https://dev.azure.com/linq2db/linq2db/_packaging?_a=package&feed=linq2db%40Local&package=linq2db.LINQPad&protocolType=NuGet).
 
-Latest build is hosted on [AppVeyor](https://ci.appveyor.com/project/igor-tkachev/linq2db-linqpad/build/artifacts).
+## Installation
 
-## Installation:
+### LINQPad 5 (from file)
 
 - Download latest **.lpx** file from the link provided above.
 - Click "Add connection" in LINQPad.
 - In the "Choose Data Context" dialog, press the "View more drivers..." button.
 - In the "Choose a Driver" dialog, press "Browse..." button.
-- Select the downloaded "linq2db.LINQPad.lpx" and click "Open" button.
+- Select the downloaded file and click "Open" button.
+- In the "Choose Data Context" dialog, select "LINQ to DB" driver and click the next button.
+- In the "LINQ to DB connection" dialog, supply your connection information.
+- You're done.
+
+### LINQPad 5 (latest version)

```

Comment:
* the automatic versions should be before the manual versions for both 5 and 6
* v6 should be before v5 (since it's more recent and more active